woocommerce/woocommerce-admin
# Add missing prompts to installed extensions

Fixes #4056

We are adding support for more extensions than I'd originally thought to the MVP - miss understanding on my part.

Adds...
* AutomateWoo
* Google Ads For WooCommerce
* Hubspot for WooCommerce
* Amazon and eBay Integration for WooCommerce 

### Screenshots

![Screen Shot 2020-04-04 at 11 47 16 am](https://user-images.githubusercontent.com/355014/78415601-41c10080-766a-11ea-9d04-a71484a8bf98.png)

### Detailed test instructions:

* On your test site make sure you don't have any of the 4 plugins installed/activated
* Go to Marketing Hub page and see recommended extensions card populated.
* Make your way through installing each plugin see that is removed from recommended once installed then appears under "Installed Extensions" (and needs to be configured)
* For bonus points go through the "Configuration steps" offered by the plugins (AW doesn't have a configuration step) and confirm that the Installed Extensions updates to reflect the correct status once configured.

Note - the hubspot integration is yet to be made available on WCCOM so it will still appear recommended due to placeholder information needing to be updated (which will be done)
